corehq/apps/domain/views.py

```diff
@@ -1552,18 +1606,34 @@ def post(self, request, *args, **kwargs):
             is_saved = self.billing_account_info_form.save()
             software_plan_name = DESC_BY_EDITION[self.selected_plan_version.plan.edition]['name']
             if not is_saved:
+                downgrade_date = self.current_subscription.next_subscription.date_start.strftime(USER_DATE_FORMAT)
                 messages.error(
                     request, _(
-                        "It appears there was an issue subscribing your project to the %s Software Plan. You "
-                        "may try resubmitting, but if that doesn't work, rest assured someone will be "
-                        "contacting you shortly."
-                    ) % software_plan_name
+                        "You have already scheduled a downgrade to the %s Software Plan on %s. If this is a "
+                        "mistake, please reach out to billing-support@dimagi.com."
+                    ) % (software_plan_name, downgrade_date)
                 )
             else:
-                messages.success(
-                    request, _(
-                        "Your project has been successfully subscribed to the %s Software Plan."
+                if self.current_subscription.next_subscription is not None:
+                    # New subscription has been scheduled for the future
+                    current_subscription = self.current_subscription.plan_version.plan.edition
+                    start_date = self.current_subscription.next_subscription.date_start.strftime(USER_DATE_FORMAT)
+                    # You have successfully scheduled your current Advanced Edition Plan
+                    # subscription to downgrade to the Pro Edition Plan on Sep 19, 2018.
+                    message = _(
+                        "You have successfully scheduled a subscription to the %s Software Plan, "
+                        "set to start on %s."
+                    ) % (software_plan_name, start_date)
```

Comment:
Should there be another `if` clause in here?  Looks like `message` is overridden just below

Reply:
Nope! I just forgot to delete the original message. Thanks for catching that!